ui/hooks/experiences/useRamps.ts

```diff
@@ -13,7 +14,7 @@ const portfolioUrl = process.env.PORTFOLIO_URL;
 const useRamps = (): IUseRamps => {
   const chainId = useSelector(getCurrentChainId);
 
-  const getBuyURI = useCallback((_chainId: ChainId) => {
+  const getBuyURI = useCallback((_chainId: Hex) => {
```

Comment:
Previously the `getCurrentChainId` returned a type of `any`. But the type import added on `getProviderConfig` in this PR resulted in the `chainId` type being applied here, where it conflicted with this type.

The `ChainId` type is for _built-in_ chain IDs, so it was never correct to use it here.